index.html

```diff
@@ -1,12 +1,13 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <title>Most Awesome Landing Page You've Ever Seen</title>
-    <link rel="stylesheet" href="css/style.css">
-  </head>
-
-  <body>
+<html lang="zxx">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Most Awesome Landing Page You've Ever Seen</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
 
     <header id="header">
```

Comment:
For better website loading times, place JS <script> tags right before the closing <body> tag in your markup. 💯